projects/rescuechi/petfinder/petfinder_eda.ipynb

```diff
@@ -0,0 +1,1980 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Petfinder EDA\n",
+    "\n",
+    "Basic EDA on adopted dogs data returned from the Petfinder API."
```

Comment:
Since this notebook is still using the raw data as the input and not the cleaned file, maybe note that? or we could update the file with the cleaned data and then edit to remove some of the data cleaning that was done in the notebook.

Reply:
Updated!